Documentation/rkt_guide.md

```diff
@@ -0,0 +1,36 @@
+# Running fleetd under rkt
+
+The following guide will show you how to run fleetd under rkt.
+
+## Building fleet ACI
+
+Just run the [build-aci](/build-aci) script.
+
+## Running fleet ACI
+
+You'll need a `/run/fleet/units` directory in the host since this is where fleet stores systemd units and it will be mounted as a volume inside the container.
+
+Then you can run fleet under rkt:
+
+```
+# rkt -insecure-skip-verify run -inherit-env -volume machine-id,kind=host,source=/etc/machine-id -volume dbus-socket,kind=host,source=/run/dbus/system_bus_socket -volume fleet-units,kind=host,source=/run/fleet/units -volume etc-fleet,kind=host,source=/etc/fleet fleetd-0.9.0.aci
```

Comment:
Pointing to that would be fine. I was confused because I didn't any environment variables defined in the systemd unit file below so `-inherit-env` does not seem to do anything.